docs/developer-guide/system-tests.md

```diff
@@ -1,6 +1,6 @@
 # Writing system tests
 
-System tests verify that Stylelint works as expected. They are another line of defense against regressions, after the unit tests and integration tests.
+System tests verify that Stylelint works as expected. They are another line of defence against regressions, after the unit tests and integration tests.
```

Comment:
`defense` is a valid American English AFAIK.

Reply:
@JounQin Sorry about that. Corrected it again.